content/SCALE/SCALETutorials/Apps/AppSecurity/CloudflareTunnel.md

```diff
@@ -0,0 +1,86 @@
+## Setting Up Cloudflare
+
+Register or log in to a [Cloudflare account](https://dash.cloudflare.com/sign-up).
+A free account is sufficient.
+
+Follow Cloudflare documentation to [register a domain](https://developers.cloudflare.com/registrar/) and set up [DNS](https://developers.cloudflare.com/dns/).
+
+### Creating a Cloudflare Tunnel
+
+[Cloudflare Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/) is a client that runs on the TrueNAS SCALE system and proxies traffic from the Cloudflare network to the application.
+
+{{< trueimage src="/images/SCALE/Apps/CloudflareTunnelOverview.jpg" alt="Cloudflare Tunnel Overview" id="Cloudflare Tunnel Overview" >}}
+<p style="text-align: center;"><small> Illustration via <a href="https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/">Cloudflare</a> (CC BY) </small></p>
```

Comment:
I ended up adding `caption` support to the trueimage shortcode (thanks for the idea) so you can just use this now:

```suggestion
{{< trueimage src="/images/SCALE/Apps/CloudflareTunnelOverview.jpg" alt="Cloudflare Tunnel Overview" id="Cloudflare Tunnel Overview" caption="Illustration via [Cloudflare](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/) (CC BY)" >}}
```